src/cpp_common/undirectedHasCostBG.cpp

```
@@ -32,7 +32,8 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #include <deque>
 #include <string>
 #include <limits>
-
+#include<cassert>
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)